### PR TITLE
Add lines parameter to prevent too high window

### PR DESCRIPTION
**Pull request by Leonardiae**
When COLS is set to a value but LINES is omitted the window starts with a height equal to the height of the desktop including the scrollbar. But because the window never starts exactly start on top, the lower part of the window is out of sight at the bottom of the desktop. After typing a few commands the cursor goes out of sight. In practice this means a user always needs to resize the windows after start. Therefore I propose a change.

If you also set LINES, for example to LINES=40, the height of the windows is what it should be but the scrollbar is lost. I guess this is the reason that the LINES parameter is omitted. 

If you delete the MODE CON setting, like I propose, then the copyright and license information goes out of sight very fast after starting the NodeCore CLI. Maybe this was the reason this setting was added in the beginning. If this is an issue then I propose to also restructure the text which is shown after NodeCli is started. For example how to connect to NodeCore with connect 127.0.0.1:10500 is shown 4-6 times (depending on if a running NodeCore is found or not) and the instruction how to start the CPU miner maybe can be deleted for example.

### CODE REVIEW

File: nodecore-cli/windowsStartScript.txt [hunk 1]
```diff
@@ -1,4 +1,3 @@
-mode con: cols=90
```
Comment (PCasafont) [inline nodecore-cli/windowsStartScript.txt]: You just deleted the mode line...
Maybe add a lines parameter as you just said in the PR title, such as:
`mode con: lines=50 cols=90`

Comment (Leonardiae) [reply nodecore-cli/windowsStartScript.txt]: That's an option but then you loose the scrollbar.

**Comment by PCasafont** [comment]
Closed for now, apologies

**Comment by Leonardiae** [comment]
No problem, thanks.

**Pull request closed without merging**